config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   mount Sidekiq::Web, at: 'sidekiq'
+  mount PgHero::Engine, at: 'pghero'
```

Comment:
This is new

Reply:
![image](https://user-images.githubusercontent.com/28274491/143238823-45ba5ca1-c6da-43ef-bcb2-efcffe131583.png)